### DISCUSSION TITLE
How to avoid a jumping input/layout on Error?

### DISCUSSION THREAD

**Discussion by hensansi** [mantinedev/mantine#3203] [Q&A] [answered]
So I'm trying to avoid the inputs to jump on error but the only way I found/thought about was by creating an hidden spacing Error component. 

Is there a way to accomplish this on Mantine without creating a new component or by using the theme to fill in the space? I can't style the error because if there is no error the error component isn't present.

https://codesandbox.io/s/error-jumping-bd189q?file=/src/App.tsx
(Click on the Continue button to see the jumping happening)

**Comment by rtivital** [comment, marked as answer]
Add absolute position to error – https://codesandbox.io/s/error-jumping-forked-rf62ri?file=/src/App.tsx

**Reply by hensansi** [reply]
I checked your solution and it still jumps but I adapted it to fit what I want. 

The issue with the margin is that it isn't added by the error but by the wrapper when there is an error. The solution for that is to set the margin and then it doesn't jump.

https://codesandbox.io/s/error-jumping-fixed-szvx03

Thank you for the help, it never crossed my mind that I could make the error's position absolute.

The final generic solution (for futher reference):
```
theme={{
  components: {
    InputWrapper: {
      styles: {
        error: {
          position: "absolute"
        }
      }
    },
    Input: {
      styles: {
        wrapper: {
          marginBottom: "5px"
        }
    }}
  }
}}
      ```
